### PR TITLE
 Add auction partner name, estimate and premium

### PR DESCRIPTION
**Pull request by oxaudo**
This still needs a mediator event in Force for auction Buyer's premium.

Or it might be worth to just rebuild this modal here in reaction. But seems like eigen is also still using Force html page for that, so I figured - I'll go with a mediator call for now.

@cavvia - also adding analytics call for the 'buyer's premium' click. Figured it's not going to hurt to track it.

### CODE REVIEW

File: src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.test.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  describe("ArtworkSidebarAuctionPartnerInfo", () => {
+    beforeEach(() => {
+      artwork = Object.assign({}, ArtworkWithEstimateAndPremium)
+    })
+
+    it("displays partner name, estimate and premium", async () => {
+      const wrapper = await getWrapper(artwork)
+
+      expect(wrapper.text()).toContain(artwork.partner.name)
```
Comment (damassi) [inline src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.test.tsx]: I would avoid writing tests that are bound to fixtures -- its slightly safer (and easier to grok) working with plain strings.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 1674b666896410a00b371dc14bd35c8211221f55

https://deploy-preview-1638--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.test.tsx (hunk 1 above)
Comment (oxaudo) [reply src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.test.tsx]: The string is coming from a fixture:) But will happily update!

**Comment by oxaudo** [comment]
Corrected the test and merging.

**Pull request merged**